### ISSUE TITLE
Home view doesn't populate

### ISSUE THREAD

**Issue by junetried** [vauvenal5/yaga#167]
I'm using the F-Droid release 0.26.1. Two months ago, ish, my home view stopped populating with new images. The ones that weren't already on my device would also never load though the home view. When I logged out and logged back in, all the images it did have saved from Nextcloud were now considered local to the device. Despite this, the browse view still worked fine. I've tried to reinstall the app and now *none* of the previously saved images are visible, the home view is empty. On the Google Play release, the browse view is broken too, it's a grey box.

I suspect this is actually an SSL issue somehow. I'm using a domain local to my home network with a self-signed certificate, but it has been added to my system, and both the Nextcloud app and Firefox (after configuring it to use the system store) will happily use my certificate. I have recently switched to Caddy from Nginx, although I didn't want to attempt to move my configuration over, so Caddy simply proxies to Nginx.

Although I didn't see anything interesting, here are the logs, when logging in:
[yaga.log.txt](https://github.com/vauvenal5/yaga/files/13696325/yaga.log.txt)
[yaga.isolate.log.txt](https://github.com/vauvenal5/yaga/files/13696326/yaga.isolate.log.txt)

**Comment by vauvenal5** [comment]
Hi! Thx for reporting. I have not updated the F-Droid version for quiet some time. I hope I can do it in the coming week. Quiet some things have changed. I will ping you once I have updated.

**Comment by junetried** [comment]
> Hi! Thx for reporting. I have not updated the F-Droid version for quiet some time. I hope I can do it in the coming week. Quiet some things have changed. I will ping you once I have updated.

I did notice that. Should I download the Google Play release and send logs from there? It doesn't seem to work there either.

**Comment by vauvenal5** [comment]
Yes, it would be helpful to see if you have the issue also on the newest version.

**Comment by comomberchese** [comment]
I'm looking forward to the new release. Thank you for your work.

**Comment by vauvenal5** [comment]
@junetried do you still have the issue?

@comomberchese do you have the same problem or are looking forward to a new F-Droid release?

**Comment by junetried** [comment]
> @junetried do you still have the issue?
> 
> @comomberchese do you have the same problem or are looking forward to a new F-Droid release?

I'm sorry, I forgot to come back with logs from the newer version. Yes, I still have the issue.

**Comment by junetried** [comment]
Here are the Yaga logs from the Google Play release:
[yaga.log.txt](https://github.com/vauvenal5/yaga/files/13849363/yaga.log.txt)
[yaga.isolate.log.txt](https://github.com/vauvenal5/yaga/files/13849364/yaga.isolate.log.txt)

**Comment by comomberchese** [comment]
> @comomberchese do you have the same problem or are looking forward to a new F-Droid release?

F-Droid

**Comment by vauvenal5** [comment]
@junetried since I did not find anything in the logs, here is what I understood so far:

- you can login successfully
- after login the home view stays empty
- the browser view works properly(?)
- are you still on the FDroid version or on the newer GPlay version?

**Comment by dz4k** [comment]
I am experiencing the same issue.

- Yes
- Yes
- Yes
- The issue occurs on both.

[yaga.log.txt](https://github.com/vauvenal5/yaga/files/14705961/yaga.log.txt)
[yaga.isolate.log.txt](https://github.com/vauvenal5/yaga/files/14705962/yaga.isolate.log.txt)
